topics/single-cell/tutorials/scrna-case_FilterPlotandExploreRStudio-WIP/preamble.md

```diff
@@ -0,0 +1,78 @@
+# Introduction
+
+You’ve previously done all the work to make a single cell matrix. Now it’s time to fully process our data using Seurat: remove low quality cells, reduce the many dimensions of data that make it difficult to work with, and ultimately try to define clusters and find some biological meaning and insights! There are many packages for analysing single cell data - Seurat (Satija et al. 2015), Scanpy (Wolf et al. 2018), Monocle (Trapnell et al. 2014), Scater (McCarthy et al. 2017), and many more. We’re working with Seurat in RStudio because it is well updated, broadly used, and highly trusted within the field of bioinformatics.
+
+> <comment-title></comment-title>
+> This tutorial is significantly based on the [Seurat documentation](https://satijalab.org/seurat) as well as [Seurat's Guided Clustering Tutorial](https://satijalab.org/seurat/articles/pbmc3k_tutorial.html).
+{: .comment}
+
+> <agenda-title></agenda-title>
+>
+> In this tutorial, we will cover:
+>
+> 1. TOC
+> {:toc}
+>
+{: .agenda}
+
+We’ll provided you with experimental data to analyse from a mouse dataset of fetal growth restriction [Bacon et al. 2018](https://usegalaxy.eu/training-material/topics/single-cell/tutorials/scrna-case_basic-pipeline/tutorial.html#Bacon2018). This is the full dataset generated from [this tutorial](https://usegalaxy.eu/training-material/topics/single-cell/tutorials/scrna-case_alevin/tutorial.html). 
```

Comment:
```suggestion
We’ll provided you with experimental data to analyse from a mouse dataset of fetal growth restriction [Bacon et al. 2018]({% cite Bacon2018 %}). This is the full dataset generated from [this tutorial]({% link topics/single-cell/tutorials/scrna-case_alevin/tutorial.html %}). 
```

please use the `{% link topics/../../ %}` syntax for links to training materials.

And if you want to link to a reference/citation, please include it in this tutorial (so add a `tutorial.bib` file and put the citation in there (in this case you can copy it from the other tutorial's bib file)